.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
Since it's not related to project, I believe it should be in `.gitignore_global` file, not in local repo.